src/client/Microsoft.Identity.Client/Internal/Requests/Interactive/AuthCodeRequestComponent.cs

```diff
@@ -60,7 +60,7 @@ public async Task<Uri> GetAuthorizationUriWithPkceAsync(string codeVerifier, Can
             return result.Item1;
```

Comment:
I think what @pmaytak is saying is that the old usages of the tuple (with .Item1 etc.) should be fixed as well. Probably just a serach for `.Item1`